run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+cd /usr/src/app
+case ${DOCKER_STATE} in
+migrate)
+    echo "running migrate"
+    bundle exec rake db:migrate
+    ;;
+seed)
+    echo "running seed"
+    bundle exec rake db:migrate
+    bundle exec rake db:seed
+    ;;
+vagrant)
+    echo "running seed"
+    bundle exec rake db:create
+    bundle exec rake db:seed
+    bundle exec rake db:migrate
+    ;;
+create)
+    echo "running seed"
+    bundle exec rake db:create
+    bundle exec rake db:migrate
+    bundle exec rake db:seed
+    ;;
+esac
+bundle exec rails s
```

Comment:
Unicorn gem is installed via bundler but never used except of testing. As it's "just" admin panel it's safe to assume that load won't be that massive and it's safe to use built-in server. Just my opinion :)